src/main/java/com/google/cloud/tools/appengine/experimental/deploy/DeployExample.java

```diff
@@ -19,13 +19,16 @@
 import com.google.cloud.tools.appengine.api.deploy.DefaultDeployConfiguration;
 import com.google.cloud.tools.appengine.cloudsdk.PathResolver;
 import com.google.cloud.tools.appengine.experimental.AppEngineRequestFactory;
+import com.google.cloud.tools.appengine.experimental.AppEngineRequestFuture;
 import com.google.cloud.tools.appengine.experimental.AppEngineRequests;
-import com.google.cloud.tools.appengine.experimental.internal.process.io.PrintStreamOutputHandler;
 
 import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
 import java.util.Collections;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.Future;
 
 public class DeployExample {
```

Comment:
What's the purpose of this class? If it's example code it doesn't really belong in this package. It's more of a tutorial thing.